doc/interop-test-descriptions.md

```diff
@@ -887,6 +943,19 @@ payload body of size `SimpleRequest.response_size` bytes and type as appropriate
 for the `SimpleRequest.response_type`. If the server does not support the
 `response_type`, then it should fail the RPC with `INVALID_ARGUMENT`.
 
+### CompressedResponse
+[CompressedResponse]: #compressedresponse
+
+When the client sets `SimpleRequest.request_compressed_response` to true, the
+response is sent back compressed.
+
+### CompressedRequest
```

Comment:
This seems error prone. If you take an server that doesn't implement this and run the client_compressed_unary against it, it will succeed so you the only way to check that the interop tests actually do what they are supposed to is to manually check the serverside code - and that's bad. Can we redefine the behavior so that you can tell servers that recognize expect_compressed_request from the ones that do not?